### PR TITLE
feat(iac): add iac ggshield action

### PR DESCRIPTION
**Pull request by zhangmatthieu**
Move ggshield action + add iac action

### CODE REVIEW

File: actions/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+
+args=("$@")
+ggshield secret scan -v ${args[@]} ci
```
Comment (agateau-gg) [inline actions/entrypoint.sh:4]: I don't know the syntax used here for `args`. What does it bring compared to the simpler `ggshield secret scan -v "$@" ci`?

**Review by agateau-gg — COMMENTED** [review]
The file hierarchy in the PR is:

```
/
  actions  # secret action
    iac    # iac action
```

Is it possible to have this instead?

```
/
  actions
    iac    # iac action
    secret # secret action
```

There is no reason for secret to be above iac.

File: actions/entrypoint.sh (hunk 1 above)
Comment (zhangmatthieu) [reply actions/entrypoint.sh:4]: Sure, will update the file hierarchy if it doesn't make sense to have secret above iac. Unsure about the syntax. This was the working syntax in ggshield-actions, I will check

Comment (zhangmatthieu) [reply actions/entrypoint.sh]: It doesn't seem to work without the args trick. The help command is displayed

**Review by ziadghalleb — APPROVED** [review]

File: actions/entrypoint.sh (hunk 1 above)
Comment (agateau-gg) [reply actions/entrypoint.sh]: Strange, it works here. Another Bash mystery I guess.

**Review by agateau-gg — CHANGES_REQUESTED** [review]
One last thing before this can go in: Can you update the README section about the action? (https://github.com/GitGuardian/ggshield#github)

**Comment by zhangmatthieu** [comment]
> Let me know if you are okay with the last change on commit [4957fdb](https://github.com/GitGuardian/ggshield/pull/330/commits/4957fdbb141225e884cc011d42c047ead1696baf).

Each type doesn't seem really descriptive. Other than that, looks good to me, let me add the commit in the PR

**Review by agateau-gg — APPROVED** [review]
Looks good, thanks!

**Pull request merged**